src/shared/Microsoft.Git.CredentialManager/GitConfiguration.cs

```diff
@@ -3,117 +3,121 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
 using System.Text;
 
 namespace Microsoft.Git.CredentialManager
 {
     /// <summary>
     /// Invoked for each Git configuration entry during an enumeration (<see cref="IGitConfiguration.Enumerate"/>).
     /// </summary>
-    /// <param name="name">Name of the current configuration entry.</param>
-    /// <param name="value">Value of the current configuration entry.</param>
+    /// <param name="entry">Current configuration entry.</param>
     /// <returns>True to continue enumeration, false to stop enumeration.</returns>
-    public delegate bool GitConfigurationEnumerationCallback(string name, string value);
+    public delegate bool GitConfigurationEnumerationCallback(GitConfigurationEntry entry);
 
     public enum GitConfigurationLevel
     {
         All,
-        ProgramData,
         System,
-        Xdg,
```

Comment:
The `XDG_*` config level still exists...

Reply:
It still exists, but we never consider it explicitly in GCM. It'll still be read by Git when we do 'unscoped' reads.